java/ql/src/Security/CWE/CWE-297/UnsafeHostnameVerification.ql

```diff
@@ -0,0 +1,75 @@
+/**
+ * @name Disabled hostname verification
+ * @description Accepting any certificate as valid for a host allows an attacker to perform a man-in-the-middle attack.
+ * @kind path-problem
+ * @problem.severity error
+ * @precision high
+ * @id java/everything-accepting-hostname-verifier
```

Comment:
```suggestion
 * @id java/insecure-hostname-verifier
```

Reply:
Applied in c69baaa1332665c55f7188057b39d0a4a066c88e